server/integration/endpoint/src/main/java/org/infinispan/server/endpoint/subsystem/EndpointSubsystemAdd.java

```diff
@@ -41,14 +37,20 @@
 import org.jboss.msc.service.ServiceBuilder;
 import org.jboss.msc.service.ServiceName;
 
+import static org.infinispan.server.endpoint.subsystem.ModelKeys.HOTROD_CONNECTOR;
+import static org.infinispan.server.endpoint.subsystem.ModelKeys.MEMCACHED_CONNECTOR;
+import static org.infinispan.server.endpoint.subsystem.ModelKeys.REST_CONNECTOR;
+import static org.infinispan.server.endpoint.subsystem.ModelKeys.ROUTER_CONNECTOR;
```

Comment:
@wburns Good point!! We should do what Intellij mandates 😜....

As long as we have an agreement I don't care 😃